backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -273,4 +279,63 @@ public static Extension convertToTribalAffiliationExtension(String tribalAffilia
     }
     return null;
   }
+
+  public static Observation convertToObservation(
+      Result result, boolean corrected, String correctionReason) {
+    if (result != null && result.getDisease() != null) {
+      return convertToObservation(
+          result.getDisease().getLoinc(),
+          result.getDisease().getName(),
+          result.getResultLOINC(),
+          corrected,
+          correctionReason,
+          Objects.toString(result.getInternalId(), ""));
+    }
+    return null;
+  }
+
+  public static Observation convertToObservation(
+      String diseaseCode,
+      String diseaseName,
+      String resultCode,
+      boolean corrected,
+      String correctionReason,
+      String id) {
+    var observation = new Observation();
+    observation.setId(id);
+    observation.setStatus(ObservationStatus.FINAL);
```

Comment:
It could be worth leaving a comment about this status changing if the test is a correction - unless you read through `addCorrectionNote`, it's not obvious that this FINAL status isn't quite finalized 😄

Reply:
Actually, reading through `addCorrectionNote` a bit further I'd make a different recommendation - I would move this call down to that helper method and set it as the body of an `else`. That makes it a bit clearer what the actual status will be.